src/Entity/Transformer.php

```diff
@@ -162,11 +162,11 @@ public function load(SnapshotInterface $snapshot)
         $page->setRequestMethod($content['request_method']);
 
         $createdAt = new \DateTime();
-        $createdAt->setTimestamp($content['created_at']);
+        $createdAt->setTimestamp((int) $content['created_at']);
```

Comment:
For this changes, isn't it better to modify instead this other code?

https://github.com/sonata-project/SonataPageBundle/blob/4f9b962f2bb5d44175bb33e1aefc3106fbec6d59/src/Entity/Transformer.php#L111

We could do instead of `format` use `getTimestamp` and we would get a integer without needing a conversion here. WDYT?

Reply:
This would store all new entries as integers, but it would not fix all the existing entries in the database which are serialized at strings.

Perhaps we could do both. :)